src/util/message.h

```diff
@@ -12,9 +12,12 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include <cstdio>
 #include <fmt/format.h>
+#include <fmt/color.h>
 #include <util/message/format.h>
 #include <util/location.h>
 
+static bool ENABLE_COLOR = false;
```

Comment:
Can you replace this global variable with an option?

Reply:
Fixed